### DISCUSSION TITLE
Enable WIki

### DISCUSSION THREAD

**Discussion by salty2011** [ATVWGS/tenant-management-framework#21] [General]
@ATVWGS any chance we could enable the wiki pages for this project. This module is beginning to get large enough to warrant a wiki with a structured kb covering all the features in greater depth.

Would be happy to start with converting the home page into a structured wiki and then begin adding to it.

**Comment by ATVWGS** [comment]
@salty2011 I've activated the Wiki pages. Can you please check if you can edit content?

**Comment by salty2011** [comment]
@ATVWGS just testing and all looks good. Thats for that
